README.rst

```diff
@@ -0,0 +1,22 @@
+################
+GRPC-OpenTracing
```

Comment:
It doesn't matter much. I like .md because it's more standard for modern OSS projects, but whatever.